examples/qm9/qm9_dataset_exploration.ipynb

```diff
@@ -0,0 +1,558 @@
+       "<path d=\"M 176.239,79.1275 208.861,6.81818\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 208.861,6.81818 244.784,10.7729\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 244.784,10.7729 280.707,14.7277\" style=\"fill:none;fill-rule:evenodd;stroke:#FF0000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 208.861,6.81818 224.972,84.4924\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 224.972,84.4924 201.697,110.087\" style=\"fill:none;fill-rule:evenodd;stroke:#000000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<path d=\"M 201.697,110.087 178.422,135.682\" style=\"fill:none;fill-rule:evenodd;stroke:#FF0000;stroke-width:2px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1\"/>\n",
+       "<text style=\"font-size:15px;font-style:normal;font-weight:normal;fill-opacity:1;stroke:none;font-family:sans-serif;text-anchor:start;fill:#FF0000\" x=\"91.5599\" y=\"102.738\"><tspan>O</tspan></text>\n",
+       "<text style=\"font-size:15px;font-style:normal;font-weight:normal;fill-opacity:1;stroke:none;font-family:sans-serif;text-anchor:start;fill:#FF0000\" x=\"280.707\" y=\"22.9988\"><tspan>O</tspan></text>\n",
+       "<text style=\"font-size:15px;font-style:normal;font-weight:normal;fill-opacity:1;stroke:none;font-family:sans-serif;text-anchor:start;fill:#FF0000\" x=\"158.096\" y=\"150.682\"><tspan>HO</tspan></text>\n",
+       "</svg>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.SVG object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from ipywidgets import interact\n",
+    "import numpy as np\n",
+    "np.set_printoptions(precision=3, suppress=True)\n",
+    "\n",
+    "def show_dataset(index):\n",
+    "    print('index={}, SMILES={}'.format(index, dataset_smiles[index]))\n",
+    "    atom, adj, labels = dataset[index]\n",
+    "    print('atom', atom)\n",
+    "    # print('adj', adj)\n",
+    "    print('labels', labels)\n",
+    "    mol = Chem.MolFromSmiles(dataset_smiles[index])\n",
+    "    return SVG(moltosvg(mol))\n",
+    "\n",
+    "interact(show_dataset, index=(0, len(dataset) - 1, 1))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Appendix: how to save the molecule figure?\n",
+    "\n",
+    "`rdkit` provides `Draw.MolToFile` method to visualize mol instance and save it to png format."
```

Comment:
You can save molecule images in SVG with RDKit. See http://cheminformist.itmol.com/TEST/wp-content/uploads/2015/07/rdkit_moldraw2d_2.html